src/test/java/wolox/training/repositories/RepositoriesTestHelper.java

```diff
@@ -0,0 +1,202 @@
+package wolox.training.repositories;
+
+import java.util.List;
+import java.util.Optional;
+import java.util.function.BiFunction;
+import java.util.function.Function;
+import java.util.function.IntFunction;
+import javax.persistence.EntityManager;
+import org.junit.jupiter.api.Assertions;
+import org.springframework.data.repository.Repository;
+import wolox.training.models.Book;
+import wolox.training.models.User;
+
+/**
+ * Helper class for persistence testing.
+ */
+/* package */ class RepositoriesTestHelper {
+
+    /**
+     * Private constructor to avoid instantiation.
+     */
+    private RepositoriesTestHelper() {
+    }
+
+    /**
+     * Asserts that the {@code expected} and the {@code actual} {@link Book}s are the same. This
+     * method is different that {@link Assertions#assertEquals(Object, Object)} as equality is not
+     * checked using the {@link Book#equals(Object)} method, but by checking all the {@link Book}
+     * fields.
+     *
+     * @param expected The expected {@link Book}.
+     * @param actual The actual {@link Book}.
+     * @param message A message to be displayed in case the assertion fails.
+     */
+    /* package */
+    static void assertSame(final Book expected, final Book actual, final String message) {
```

Comment:
maybe separete this asserts in classes by models, clean this class.
If you have more models , this class will has alot of lines of code.

Reply:
Done. Check 6dfdc05